site/docs/components/stepped-tracker/usage.mdx

```diff
@@ -22,26 +22,26 @@ data:
 
 ## Content
 
-Stepped tracker labels never truncate, only wrap. Therefore, ensure they're short and self-explanatory.
+Step labels never truncate, only wrap. Therefore, ensure they're short and self-explanatory.
 
 ## Import
 
 To import `SteppedTracker` and related components from the lab Salt package, use:
 
 ```
-import { SteppedTracker, TrackerStep, StepLabel } from "@salt-ds/lab";
+import { SteppedTracker, Step, useStepReducer } from "@salt-ds/lab";
 ```
 
 ## Props
 
 ### `SteppedTracker`
 
-<PropsTable packageName="lab" componentName="SteppedTracker" />
+<PropsTable packageName="lab" componentName="stepped-tracker" />
 
-### `TrackerStep`
+### `Step`
 
-<PropsTable packageName="lab" componentName="TrackerStep" />
+<PropsTable packageName="lab" componentName="Step" />
 
-### `StepLabel`
+### `useStepReducer`
 
-<PropsTable packageName="lab" componentName="StepLabel" />
+<PropsTable packageName="lab" componentName="useStepReducer" />
```

Comment:
blank on site

Reply:
I don't think the hook is fixible, have to look at the way types are extracted, docgen etc., and I don't think its the focus of this PR.